Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -317,7 +324,7 @@ static String defaultWebsiteDataStoreRootDirectory()
     }
 #endif
 
-    return cacheDirectoryFileSystemRepresentation("OfflineWebApplicationCache"_s, { }, ShouldCreateDirectory::No);
+    return cacheDirectoryFileSystemRepresentation("OfflineWebApplicationCacheTest"_s, { }, ShouldCreateDirectory::No);
```

Comment:
Huh?

Reply:
Oops used for testing... will remove this part

Reply:
Should likely drop this change.